example/Caddyfile

```diff
@@ -3,8 +3,10 @@
 	auto_https off
 	order coraza_waf first
 	log {
+		output stdout
 		format console
-		level info
+		level debug
```

Comment:
Makes the example more usable for debugging purposes. We were setting SecDebugLogLevel 9 but then printing only level info.